src/actions/users.js

```diff
@@ -323,12 +330,12 @@ export function getMissingProfilesByIds(userIds) {
     };
 }
 
-export function getMissingProfilesByUsernames(usernames) {
+export function getMissingProfilesByUsernames(usernames: Array<string>): ActionFunc {
     return async (dispatch, getState) => {
         const {profiles} = getState().entities.users;
 
         const usernameProfiles = Object.values(profiles).reduce((acc, profile) => {
```

Comment:
```
Cannot cast profile to UserProfile because inexact mixed [1] is incompatible with exact UserProfile [2].

     src/actions/users.js
     335│         const {profiles} = getState().entities.users;
     336│
     337│         const usernameProfiles = Object.values(profiles).reduce((acc, profile) => {
 [2] 338│             acc[(profile: UserProfile).username] = profile;
     339│             return acc;
     340│         }, {});
     341│

     /tmp/flow/flowlib_1bc5a954/core.js
 [1]  62│     static values(object: any): Array<mixed>;
```

I don't think this is wrong...

Any workaround for Object.values()?